.github/workflows/build_and_test.yml

```diff
@@ -28,11 +28,12 @@ jobs:
         java-version: '11.0.6' # The JDK version to make available on the path.
         java-package: jdk
         architecture: x64
-    - uses: ros-tooling/setup-ros@3b3f060
+    - uses: ros-tooling/setup-ros@b1c8132
```

Comment:
Looks like we can use the release tag [0.0.24](https://github.com/ros-tooling/setup-ros/tree/0.0.24).

Reply:
Fixed in 406844e.